preprocessy/encoding/_categorical.py

```diff
@@ -15,6 +15,9 @@ def __init__(self):
         self.ord_dict = None
         self.one_hot = False
 
+    def __repr__(self):
+        return f"EncodeData(target_label={self.target_label} ,train_df=None, test_df=None, cat_cols=None, ord_dict=None, one_hot={self.one_hot})"
```

Comment:
if we use self.df or self.test_df or other such params, the actual dataframe gets printed and hence must be avoided

Reply:
Since printing the actual values for these params isn't chaotic and rather precise, letting them stay as that seems fine. Printing DFs doesn't make sense, so set them to none. Thoughts?

Reply:
Yes, thats right. 
Marking the changes requested as resolved.